scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/common/util/SchedulerProxyUserInterface.java

```diff
@@ -598,4 +601,25 @@ public void putThirdPartyCredential(String key, String value) throws NotConnecte
     public void removeThirdPartyCredential(String key) throws NotConnectedException, PermissionException {
         uischeduler.removeThirdPartyCredential(key);
     }
+
+    @Override
+    public Page<TaskId> getTaskIds(String taskTag, long from, long to, boolean mytasks, boolean running,
+            boolean pending, boolean finished, int offset, int limit)
+                    throws NotConnectedException, PermissionException {
+        return uischeduler.getTaskIds(taskTag, from, to, mytasks, running, pending, finished, offset, limit);
+    }
+
+    @Override
+    public Page<TaskState>  getTaskStates(String taskTag, long from, long to, boolean mytasks,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)